src/Microsoft.Health.Dicom.Core/Features/Context/IDicomRequestContext.cs

```diff
@@ -20,5 +20,7 @@ public interface IDicomRequestContext : IRequestContext
 
     long BytesTranscoded { get; set; }
 
+    int RequestParts { get; set; }
```

Comment:
How exactly will this be used? Somewhere we'll wait for each of the parts to be read and then pass this information as a billing metric?

Reply:
In `BillingMiddleware.cs` in workspace-platform repo. It already reads some other fields from the request context (like BytesTranscoded), so now it will also read RequestParts and use that to bill for ApiRequests